package/MDAnalysis/coordinates/base.py

```diff
@@ -141,9 +141,15 @@
 from ..auxiliary.core import get_auxreader_for
 
 class Namespace(object):
-    """Empty class 
-    To allow storing attributes in new namespace """
-    pass
+    """Class to allow storing attributes in new namespace. """
```

Comment:
As @richardjgowers suggested, you could inherit from `dict`. This would allows things like:

```python
a = Namespace()
a.toto = 3
b = a['toto']
del a['toto']
```

As ou can see, it would avoid direct calls to `__dict__` from outside the `Namespace` class.